jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/ResourcesAnnotationTest.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
```

Comment:
Since this is a new file the first like should be:
Copyright (c) 2024 IBM Corporation and others.